includes/wikia/models/NavigationModel.class.php

```diff
@@ -551,6 +551,8 @@ public function parseOneLine($line) {
 		if ( empty( $doNotLink ) ) {
 			if ( preg_match( '/^(?:' . wfUrlProtocols() . ')/', $link ) ) {
 				$href = $link;
+			} else if ( preg_match( '/^\/{1}\w+[\w\/]*/', $link ) ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This branch's code block is the same as the block for the branch on line 552. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1871)